### DISCUSSION TITLE
Option for not sending copy of signed document to the signee

### DISCUSSION THREAD

**Discussion by matbe81** [docusealco/docuseal#270] [Ideas]
As an option

**Comment by artpumpkin** [comment]
Is this possible please? So we can customize the "document copy" email as we wish.

**Comment by omohokcoj** [comment]
@artpumpkin it's possible to disable this email on the template preferences level
![image](https://github.com/user-attachments/assets/4330d3b3-1665-4936-abed-8a41d8e921b6)

**Reply by artpumpkin** [reply]
Hey @omohokcoj, appreciate the response, unfortunately I don't have this option not sure why.

<img width="588" alt="image" src="https://github.com/user-attachments/assets/5118d413-2607-4efe-98d2-5b6c882aa9f0">

**Reply by omohokcoj** [reply]
@artpumpkin update to the latest version

**Reply by artpumpkin** [reply]
Hey @omohokcoj, how to do it please, I'm using docuseal dashboard, cloud version.

<img width="915" alt="image" src="https://github.com/user-attachments/assets/e560355b-49e6-4c56-b1ca-bc402eaeb800">

**Reply by omohokcoj** [reply]
@artpumpkin oh understood - it's not under personalization settings - but under template preferences:
![image](https://github.com/user-attachments/assets/d6c9a9e1-fa80-4efc-b2c0-f1ecb2becc1b)
![image](https://github.com/user-attachments/assets/74f928a6-229f-45d9-9ffb-dde1b00725cc)

**Reply by artpumpkin** [reply]
Hey @omohokcoj, thank you so much man, I have found it.
